src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/WorkloadResolver.cs

```diff
@@ -561,6 +561,11 @@ public IWorkloadResolver CreateOverlayResolver(IWorkloadManifestProvider overlay
             return overlayResolver;
         }
 
+        public string GetSdkFeatureBand()
+        {
+            return _manifestProvider?.GetSdkFeatureBand() ?? string.Empty;
```

Comment:
Is it ever correct to return an empty string, or should this throw an Exception in that case?